server/game/cards/12-2T2D/AlexanderSequoia.js

```diff
@@ -0,0 +1,30 @@
+const GameActions = require('../../GameActions/index.js');
+const DudeCard = require('../../dudecard.js');
+
+class AlexanderSequoia extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.isInTownSquare(),
+            effect: ability.effects.modifySundownDiscard(1)
+        });
+        
+        this.action({
+            title: 'Noon',
```

Comment:
Use at least the card name, or some short description as Title, because it is possible some other action can be added to a dude (e.g. via Nicodemus Whateley), and then the title is used in the menu and "Noon" is not very good.